service/src/main/resources/application.properties

```diff
@@ -166,3 +166,7 @@ quarkus.log.category."io.fabric8.kubernetes.client.informers.cache".level=WARN
 %test.app.ldap.search.filter=testid=%s
 
 %test.quarkus.log.category."io.spaship".level=DEBUG
+
+event.sourcing.enabled=${ES_ENABLED:true}
```

Comment:
It's not needed to use "ES_ENABLED" env variable because EVENT_SOURCING_ENABLED is the env variable consumed by Quarkus. Simply use `event.sourcing.enabled=true`.
Please mention this property in docs: https://github.com/spaship/operator/blob/main/docs/operator-manual/configuration.md which is then available on https://spaship.io/operator/operator-manual/configuration/